Uzira.rte/Craft/Summoning/Summoning.lua

```diff
@@ -0,0 +1,34 @@
+function Create(self)
+	local largestRadius = 0;
+	for actor in self.Inventory do
+		if IsActor(actor) then
+			actor = ToActor(actor);
+			if actor.ModuleName == "Uzira.rte" then
+				actor:AddScript("Uzira.rte/Actors/Shared/Undead.lua");
+			else
+				actor.HUDVisible = false;
+				actor.Health = 0;
```

Comment:
I'm guessing you can't just set them ToDelete = true?
Or is your goal to make them show up as dead bodies? If it is, it's not working, all I get is a death sound but no body.

Reply:
That's because their body is underground lmao

Reply:
Haha I thought it might be